boards/nxp/frdm_ke17z/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+# FRDM-KE17Z board
+
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_FRDM_KE17Z
+
+config FXOS8700_DRDY_INT1
+	default y
```

Comment:
this is DNP in revB board, so we should not enable it

Reply:
Thanks for checking and testing, I will drop the sensor related configuration